.devcontainer/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:18.04
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+# Set the default shell to bash rather than sh
+ENV SHELL /bin/bash
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils sudo 2>&1 \
+    #
+    # Verify git, process tools, lsb-release (common in install instructions for CLIs) installed
+    && apt-get -y install git procps lsb-release software-properties-common \
+    #
+    # Install Docker CE CLI
+    && apt-get install -y apt-transport-https ca-certificates curl gnupg-agent \
+    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get -y install docker-ce \
+    #   #
+    # Java install for FxA
+    && apt-get -y install openjdk-11-jre \
+    #    #
+    # FxA dependencies.
+    && apt-get -y install build-essential git-core libgmp3-dev graphicsmagick python-virtualenv python-dev pkg-config libssl-dev \
```

Comment:
`git-core` is a virtual package:

> shane@shane-VirtualBox:~/development/fxa$ apt show git-core
Package: git-core
State: not a real package (virtual)
N: Can't select candidate version from package git-core as it has no candidate
N: Can't select versions from package 'git-core' as it is purely virtual
N: No packages found

Does it install correctly? When I tried that command line in a new ubuntu VM it failed and I had to install `git` instead.

Reply:
It did seem to install correctly, I'll change it to git and retry to verify that's fine as well.